src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -44,12 +44,21 @@
 import static uk.gov.hmcts.reform.sscs.model.hmc.reference.DayOfWeekUnavailabilityType.ALL_DAY;
 import static uk.gov.hmcts.reform.sscs.model.hmc.reference.PartyType.INDIVIDUAL;
 import static uk.gov.hmcts.reform.sscs.model.hmc.reference.PartyType.ORGANISATION;
+import static uk.gov.hmcts.reform.sscs.reference.data.model.EntityRoleCode.REPRESENTATIVE;
 import static uk.gov.hmcts.reform.sscs.reference.data.model.EntityRoleCode.RESPONDENT;
 
 @SuppressWarnings({"PMD.GodClass", "PMD.ExcessiveImports", "PMD.TooManyMethods"})
 // TODO Unsuppress in future
 public final class HearingsPartiesMapping {
 
+    /**
+     * According to task <a href ="https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=2115&projectKey=SSCS&view=detail&selectedIssue=SSCS-10747">SSCS-10747</a>
+     * for representatives it is asked to return solicitor code as the relationship type
+     * which is 11 that is why this constant is defined.
+     * It is value is defined as 11.
+     */
+    public static final String REPRESENTATIVE_RELATION_TYPE_SOLICITOR_CODE = "11";
```

Comment:
Can you use/update/append this to gov.hmcts.reform.sscs.reference.data.model.EntityRoleCode instead of creating a constant. This should be under the enum LEGAL_REPRESENTATIVE if I am not mistaken.

Reply:
That code is RPTT and reason why this task is opened was replacing RPTT with 11 which is SOLICITOR code. Issue is https://tools.hmcts.net/jira/browse/SSCS-10747. It is mentioned by Sean that code for Representative should always be replaced with 11. I had that noted in the issue.